ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -61,6 +61,8 @@ service PartyManagementService {
   // canton: completely different globally unique identifier is allocated.
   // Behind the scenes calls to an internal protocol are made. As that protocol
   // is richer than the surface protocol, the arguments take implicit values
+  // The party identifier suggestion must be a valid party name. Party names are required to be non-empty US-ASCII strings built from letters, digits, space,
```

Comment:
is this actually true in general? I know it applies to Sandbox because it literally tries to use the hint as the party id but does it apply to canton, corda, …?

Reply:
oh nvm, I missed the code below. Seems fine to me then.